src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -240,13 +246,19 @@ const EditTestCase = () => {
     });
   }, [formik.dirty, currentTestCase?.json]);
 
+  const discardChanges = () => {
+    resetForm();
+    setDiscardDialogOpen(false);
+  };
```

Comment:
Looks like our Test coverage is down, can we add a test case to cover discard changes scenarios ?